packages/atomic/src/images/clock.svg

```diff
@@ -1 +1 @@
-<svg fill="none" stroke="currentColor" stroke-linejoin="round" stroke-linecap="round"  viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="5.5"/><path d="m8.5 4.5v4"/><path d="m10.3066 10.1387-1.80932-1.5768"/></svg>
\ No newline at end of file
+<svg fill="none" stroke="currentColor" stroke-linejoin="round" stroke-linecap="round"  viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><circle cx="8" cy="8" r="7.5"/><path d="m8.5 4.5v4"/><path d="m10.3066 10.1387-1.80932-1.5768"/></svg>
```

Comment:
What was wrong with the SVG ?

Reply:
Too small vs the parent atomic-icon. Changed `r="7.5"` which allowed me to remove the `-ml-1` and use the same size as the other icon (magnifier).